Observer/Order/SaveOrderExtensionDataObserver.php

```diff
@@ -76,10 +74,6 @@ public function execute(Observer $observer)
         $orderExtensionData = $this->orderExtensionDataFactory->create();
         $orderExtensionData->setOrderId($orderId);
         $orderExtensionData->setPublicId($publicOrderId);
-        $this->eventManager->dispatch(
```

Comment:
As I know this event used before to save additional data to order, like data from LiFE elements. Any reason why we need to remove this?